packages/client/modules/userDashboard/components/OrgBilling/BillingForm.tsx

```diff
@@ -125,12 +126,20 @@ const BillingForm = (props: Props) => {
         setErrorMsg(newErrMsg)
         return
       }
-      const {error} = await stripe.confirmCardPayment(stripeSubscriptionClientSecret)
+      const {error, paymentIntent} = await stripe.confirmCardPayment(stripeSubscriptionClientSecret)
       if (error) {
         setErrorMsg(error.message)
         setIsLoading(false)
         return
       }
+      if (paymentIntent.status === 'succeeded') {
+        commitLocalUpdate(atmosphere, (store) => {
+          const organization = store.get(orgId)
+          if (!organization) return
+          organization.setValue('team', 'billingTier')
+          organization.setValue('team', 'tier')
+        })
+      }
```

Comment:
-1 let's put this logic in the `upgradeToTeamTierSuccessUpdater` where we also open the drawer and show the confetti